client/src/pages/Home.jsx

```diff
@@ -13,16 +13,11 @@ export default function Home() {
       <Container data-testid="home" className="homepage">
       <div className="home d-flex flex-column align-items-center justify-content-center flex-wrap text-center">
         <h1 className="home-title">Your Daily Fitness & Health Tracker</h1>
-        <p className="home-text">
-          Cardio? Resistance? BMI Check? Or both? Track your daily exercises and BMI count to stay fit!
-          with us.
-        </p>
         {currentUser ?
           (
```

Comment:
I think it would nice to have an icon in the main homepage, like we do for Fitness and BMI. But it's a just a suggestion tho!

Reply:
![image](https://github.com/user-attachments/assets/89920e72-8c42-49f7-905a-16d26cf03f70)
Updated!